GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/ProductUpgraderTests.cs

```diff
@@ -18,9 +18,9 @@ public override void Setup()
         public void UpgradeAvailableOnFastWhileOnLocalNoneRing()
         {
             this.SimulateUpgradeAvailable(

```

Comment:
Should this class also be renamed?  It appears that it's now simulating\testing GitHub upgrades specifically.

Reply:
Or if the plan is to make this generic in an upcoming PR then I think it's OK to leave the naming as-is.

Reply:
Agree. This and UpgradeTests.cs test GitHubUpgrader. Have to be renamed & consolidated. Please see my comment below.